# `Promise.reject(r)`

Funcția `reject` returnează o nouă promisiune care a fost respinsă cu argumentul care i-a fost pasat.
